### ISSUE TITLE
Add user and admin authentication

### ISSUE THREAD

**Issue by RobRoseKnows** [nisargap/vital-sines#4]
We need to add authentication for admins (to change tests) and users (to cache results). I'm going to try to implement authentication using Firebase, as they have some handy dandy tools to do so.

**Comment by ayebear** [comment]
If you use Firebase for that, you might as well move the genome tests there too. No sense in having both Parse and Firebase.

**Comment by RobRoseKnows** [comment]
Good point. But I don't have access to the Parse database. Can you add me as a collaborator? The email for my Parse account is bertra97 (at) gmail (dot) com.
